capstone/src/main/java/com/google/univiz/CollegeDataApi.java

```diff
@@ -0,0 +1,9 @@
+package com.google.univiz;
+
+import java.util.List;
+import java.util.Set;
+
+public interface CollegeDataApi {
+
+  List<CollegeData> getCollegesById(Set<String> ids, Set<String> names);
```

Comment:
Here a String is used for the college id, but in CollegeData, it's an int. Which should we use? Or should there be a dedicated `CollegeId class`?